resources/js/components/MobileMenu.vue

```diff
@@ -0,0 +1,106 @@
+<script setup>
+import { Link } from '@inertiajs/inertia-vue3'
+import { HomeIcon, LightBulbIcon, MapIcon, NewspaperIcon } from "@heroicons/vue/24/solid";
+defineProps({
+    user: Object
+})
+
+const items = [
+    {
+        name: 'Home',
+        component: 'Home',
+        url: '/',
+        icon: HomeIcon
+    },
+    {
+        name: 'Lộ trình',
+        component: 'LearningPaths',
+        url: '/learning-paths',
+        icon: MapIcon,
+    },
+    {
+        name: 'Học',
+        url: '/courses',
+        component: 'Courses',
+        icon: LightBulbIcon,
+    },
+    {
+        name: 'Blog',
+        component: 'Blog',
+        url: '/blog',
+        icon: NewspaperIcon
+    },
+]
+</script>
+
+<template>
+        <div class="fixed top-0 left-0 h-[150vh] z-21 w-full bg-[rgba(0,0,0,.3)]">
+            <div class="w-[60%] bg-white z-10 h-full absolute top-0 left-0">
+                <div class="flex flex-col text-[1.6rem] max-h-full overflow-y-auto pt-[16px] pl-[32px]">
+                    <div class="mt-6">
+                        <ul class="w-full text-[1.4rem] font-normal leading-5 mb-3 py-4 border-b">
+                            <li>
+                                <div v-if="user">
+                                    hello
+                                </div>
+                                <div v-else>
+                                    <a class="flex items-center py-5 pl-5 text-[1rem]" href="#">
```

Comment:
xoá text-1[rem], đổi thẻ `<a> đăng nhập thành

```suggestion
                                    <Link class="flex items-center py-5 pl-5" href="/login">
```